docs/api-docs/storefront/scripts-overview.mdx

```diff
@@ -106,6 +106,26 @@ Solutions that add dynamic script support using revisions to or additions of scr
 
 Release notes can work for all storefronts. Changes to the [OAuth scopes](/docs/start/authentication/api-accounts#oauth-scopes) of apps that use [app-level API accounts](/docs/start/authentication/api-accounts#app-level-api-accounts) require the store owner or other authorized user to accept the new scopes, which could provide a mechanism to trigger release note display. Store owners aren't prompted to reauthorize until they next open an app in the control panel, so direct contact may be advisable from a conversion perspective. You can't change the OAuth scopes for a [store-level API account](/docs/start/authentication/api-accounts#store-level-api-accounts), so directly contacting the store or stores that you want to change may be your only option for disclosure.
 
+## Subresource integrity
+
+[Subresource integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) is a security feature browsers use to verify that attackers have not manipulated the script. 
+
+All scripts on the checkout page need at least one SRI hash to meet [PCI 4.0 - 6.4.3 requirements](https://pcipolicies.com/blogs/news/how-to-comply-with-the-new-pci-dss-requirement-6-4-3). 
+
+You can [generate SRI hashes](https://www.srihash.org/) or work with a script provider or host to generate a hash. We support any of the algorithms (SHA-256, SHA-384, SHA-512) used to generate the hash.
+The hash corresponds to the contents of the script. You need to generate a different hash if you update the contents of your script.
+
+You can supply SRI hashes through [Script Manager](https://support.bigcommerce.com/s/article/Using-Script-Manager) or the [Scripts API](/docs/rest-management/scripts).  
+For _external SRC scripts_, BigCommerce will set the `integrity` attribute when we generate the `script` tag from your external SRC script.
+
+We support multiple hashes so that the script provider can update the script without invalidating it.
+Before the contents of the script change, you should supply another hash. The browser will check each hash to see if any of them match a corresponding script. 
+You can add up to 5 hashes or remove SRI hashes when they are no longer valid. 
```

Comment:
can you be clearer about what invalidates a script or hash?

Reply:
Just added a line to the start of the paragraph